integrations/service/node/service/clientApi.ts

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
+import { generateAccountData } from '../utils';
+
+import * as path from 'path';
+import * as dotenv from 'dotenv';
+
+dotenv.config({ path: path.resolve('../../../.env') });
+
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
+      API_ENV: 'staging' | 'demo' | 'prod'
+    }
+  }
+}
+
+const CLIENT_API_HOST = {
+  staging: 'https://api-staging.airwallex.com/api/v1',
+  demo: 'https://api-demo.airwallex.com/api/v1',
+  prod: 'https://api.airwallex.com/api/v1',
+};
+
+const clientApi: any = axios.create({
+  baseURL: CLIENT_API_HOST[process.env.API_ENV],
+  headers: {
+    'Content-Type': 'application/json',
+  },
+});
+
+export const login = async (requestBody: Record<string, any>, configs: Record<string, any>) => {
+  const { data: { token } } = await clientApi.post('/authentication/login', requestBody, configs);
+  return token;
+};
+
+export const createBusinessAccount = async (configs: Record<string, any>) => {
+  const { data: { id } } = await clientApi.post('/accounts/create', generateAccountData(), configs);
```

Comment:
We can put the create account API doc link here so that user can refer to which field they can pass for prefill.